src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
```

Comment:
You can't use `default` but you can use `@default`. Go with `defaultValue` if that is your preference. I will play with the API next week.